VERSION

```diff
@@ -1 +1 @@
-1.11.1
+1.11.2
```

Comment:
Ok, making a new PR and updating this one.

Note, that, the bump version PR would have to be merged first.